demo/src/demoApp.js

```diff
@@ -42,39 +42,40 @@ function getDefaultNavigationStyle() {
     statusBar: {
       visible: true,
       style: 'light',
-      backgroundColor: Colors.primary // for Android
+      backgroundColor: Colors.$backgroundPrimaryHeavy // for Android
     },
     layout: {
-      backgroundColor: Colors.white,
-      orientation: ['portrait', 'landscape']
+      backgroundColor: Colors.$backgroundDefault,
+      orientation: ['portrait', 'landscape'],
+      componentBackgroundColor: Colors.$backgroundDefault
```

Comment:
What's this for?

Reply:
run the demo app and toggle to dark mode (without changing the config file)

Reply:
So you prevent the color change?

Reply:
I set the right colors instead the defaults we have today